vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
I think we've been trying to keep the driver headers out of `vic_run`.  What is this needed for?